src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/model/CreateAccountRequest.java

```diff
@@ -0,0 +1,92 @@
+package ca.bc.gov.open.jag.efilingcommons.model;
+
+/**
+ * Represents a request to create an account
+ */
+public class CreateAccountRequest {
+
+    private String firstName;
+    private String lastName;
+    private String middleName;
+    private String email;
+
+    protected CreateAccountRequest(CreateAccountRequest.Builder builder) {
+
+        this.firstName = builder.firstName;
+        this.lastName = builder.lastName;
+        this.middleName = builder.middleName;
+        this.email = builder.email;
+
```

Comment:
We need the BCeID GUID here as well to link BCeID -> CSO account

Reply:
added as `universalId`